adapters/bombcrypto/index.js

```diff
@@ -0,0 +1,129 @@
+require("dotenv").config();
+
+const moment = require("moment");
+const BSC = require("../../sdk/EVMC");
+const axios = require("axios");
+const URL = "http://nft-sales-service.dappradar.com/open-source";
+const KEY = process.env.DAPPRADAR_API_KEY;
+const path = require("path");
+const BigNumber = require("bignumber.js");
+
+class BombCrypto {
+    constructor() {
+        this.name = "bombcrypto";
+        this.symbol = 'BCOIN';
+        this.token = "0x0000000000000000000000000000000000000000";
+        this.protocol = "binance-smart-chain";
+        this.block = 11103053;
+        this.contract = "0x00e1656e45f18ec6747F5a8496Fd39B50b38396D";
```

Comment:
All contracts and addresses must be lowercased.

Reply:
Made the change now.